### DISCUSSION TITLE
Issue reporting process with upstream repository

### DISCUSSION THREAD

**Discussion by Tugdwal** [microsoft/vscode-discussions#905] [Extension Development QnA] [answered]
Hi,

I have an issue with syntax highlighting I would like to report (and maybe fix myself). The issue is related to the builtin bat extension whose syntax is converted from https://github.com/mmims/language-batchfile. There are few similar issues like https://github.com/microsoft/vscode/issues/43923 which were moved/copied to the upstream repository: https://github.com/mmims/language-batchfile/issues/25.

Should I create an issue on vscode repository first (and copy/move it) or directly on the upstream repository?

The syntax file says:

> If you want to provide a fix or improvement, please create a pull request against the original repository. Once accepted there, we are happy to receive an update request.

In this case, how do I make an "update request" without the original vscode issue?

**Comment by isidorn** [comment, marked as answer]
I suggest to create an issue directly in the upstream repository.
However creating an issue in VS Code as well is not wrong. Thanks for your contribution
